### ISSUE TITLE
os.exec doesn't work with windows start command

### ISSUE THREAD

**Issue by ChildishGiant** [vlang/v#6306]
**V version:** V 0.1.29 b015033
**OS:** Windows 10

**What did you do?**
`os.exec("start C:\\Users\\allie\\Downloads\\hello_world.exe") or { panic(err) }`

**What did you expect to see?**
The same output as if I ran this in powershell or cmd (starting the path supplied). 
 
**What did you see instead?**
```
PS C:\Users\allie\Downloads> .\repro.exe
V panic: exec failed (CreateProcess) with code 2: The system cannot find the file specified.
 cmd: start C:\Users\allie\Downloads\hello_world.exe
Unhandled Exception 0xC0000005
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:5499: at print_backtrace_skipping_top_frames_tcc: Backtrace
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:5474: by print_backtrace_skipping_top_frames
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:5519: by unhandled_exception_handler
7ffac15a802c : by ???
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:5533: at break_if_debugger_attached: RUNTIME ERROR: invalid memory access
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:5289: by v_panic
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:10047: by main__main
C:/Users/allie/AppData/Local/Temp/v/repro.tmp.c:10142: by wmain
00445130 : by ???
00445293 : by ???
7ffabf8b6fd4 : by ???
```

**Comment by ntrel** [comment]
Maybe you need to include the extension assuming `start` is an executable.
BTW you can use `r"c:\path\file"` to avoid escaping.

**Comment by JalonSolov** [comment]
'start' isn't a separate command (there is no separate `start.exe`), it is a shell built-in, so you have to invoke a shell to use it.

Something like `os.exec(r"cmd /c start C:\Users\allie\Downloads\hello_world.exe") or { panic(err) }` might work, or you may need /k instead of /c.  Run `cmd /?` at a prompt to see the options.

**Comment by JalonSolov** [comment]
Of course, if you're invoking a shell anyway, you don't need the `start` anyway.  Just do

`os.exec(r"cmd /c C:\Users\allie\Downloads\hello_world.exe") or { panic(err) }`

If you really need to, although in most cases a simple

`os.exec(r"C:\Users\allie\Downloads\hello_world.exe") or { panic(err) }`

should suffice.

**Comment by ChildishGiant** [comment]
Without using a start command, a command prompt stays open. The actual exe I'm trying to launch will stay open after the script closes and using start makes this happen without a command prompt staying open.  Using the `cmd /c` or `cmd /k` trick doesn't act any differently from just running the exe so I'm back to the same issue. The raw string trick is a good tip though :)

**Comment by ChildishGiant** [comment]
I've managed to get it working! instead of `cmd /c` I needed `PowerShell.exe -windowstyle hidden start`. I still feel like surely any command ran with ox.exec should behave as if executed in the commandline but this is an okay work around for now.

**Comment by JalonSolov** [comment]
You are correct that it should work with any _real_ command.  'start' is not a _real_ command, or there would be a separate 'start.exe' on your system.  However, it is something that only exists within the Windows shell as a built-in, so you have to execute a shell to get it to work.

**Comment by ArtemkaKun** [comment]
I think this can be closed since:
1. Solution was found
2. Not really a V issue

What do you think @ChildishGiant ?
